src/features/cards/organisms/card-item.js

```diff
@@ -76,3 +86,48 @@ const CardBox = styled(Card)`
   max-height: 24rem;
   overflow-y: hidden;
 `
+
+const PopUpButton = () => {
+  const [popup, setPopup] = useState(false)
+
+  return (
+    <div>
+      <Button
+        small
+        onClick={() => {
+          setPopup(() => !popup)
+        }}
+      >
+        PopUp
+      </Button>
+      {!!popup && (
```

Comment:
You don't need `!!` here

Reply:
There was a discussion on React Telegram about that. I saw this !! in few popup examples, people said that it is reinsurance for some cases. Please confirm that I need to delete it.

Reply:
You can delete it